File: 2015/ruby/test/test_day02.rb
require 'test/unit'
require_relative '../lib/day02'

class TestDay02 < Test::Unit::TestCase
  def test_part1
    input = "
2x3x4
1x1x10"
    assert_equal 101, Day02.part1(input)
  end

  def test_part2
    input = "
2x3x4
1x1x10"
    assert_equal 48, Day02.part2(input)
  end
end

File: 2016/python/tests/test_day02.py
import unittest

from days.day02 import part1, part2


class TestDay02(unittest.TestCase):
    def test_part1(self):
        input = "ULL\nRRDDD\nLURDL\nUUUUD"
        expected = "1985"
        self.assertEqual(part1(input), expected)

    def test_part2(self):
        input = "ULL\nRRDDD\nLURDL\nUUUUD"
        expected = "5DB3"
        self.assertEqual(part2(input), expected)
